content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -238,11 +238,6 @@ Source:
 Events:                <none>
 ```
 
-Enabling the `CSIMigration` feature for a specific in-tree volume plugin will remove
```

Comment:
Should we still keep the first half of this paragraph?

`CSIMigration` feature is GA now, that means for a specific in-tree volume plugin it will remove
the `kubernetes.io/pv-controller` finalizer, while adding the `external-provisioner.volume.kubernetes.io/finalizer`
finalizer.